src/components/v5/shared/Navigation/Sidebar/sidebars/ColonyPageSidebar/partials/SidebarActionsSection.tsx

```diff
@@ -0,0 +1,26 @@
+import { FilePlus, MagicWand, Money } from '@phosphor-icons/react';
+import React from 'react';
+
+import { Action } from '~constants/actions.ts';
+
+import SidebarActionItem from '../../../partials/SidebarActionItem/index.ts';
```

Comment:
You're totally right @Nortsova! I tell you what, as soon as Dave's PR gets merged: https://github.com/JoinColony/colonyCDapp/pull/2807, this will be a thing of the past!